src/char/necro.py

```diff
@@ -0,0 +1,645 @@
+import keyboard
+from utils.custom_mouse import mouse
+from char import IChar
+from template_finder import TemplateFinder
+from ui import UiManager
+from pather import Pather
+from logger import Logger
+from screen import Screen
+from utils.misc import wait, rotate_vec, unit_vector
+import random
+from typing import Tuple
+from pather import Location, Pather
+import numpy as np
+import time
+from utils.misc import cut_roi, is_in_roi
+
+import mouse as _mouse
+from mouse import _winmouse
+
+class Necro(IChar):
+    def __init__(self, skill_hotkeys, char_config, screen: Screen, template_finder: TemplateFinder, ui_manager: UiManager, pather: Pather):
+        Logger.info("Setting up Necro")
+        super().__init__(skill_hotkeys, char_config, screen, template_finder, ui_manager)
+        self._pather = pather
+        #custom pindle paths for necro - there must be a better way?
+        self._pather._paths = {
```

Comment:
I agree, there must be a better way :D So I assume for some pathes you want to have a different order of nodes? First of all I dont think you want to change any of the town pathing, right? So doing sth like this:
```python
self._pather._paths[Location.A5_PINDLE_START, Location.A5_PINDLE_SAFE_DIST] = [100, 101]
``` 
would be enough. You could add a setter for it inorder to not access private members. similar to the offset_node() function in pather. Sth like `adapt_path(key: tuple(Location, Location), new_node_sequence: list[in])`

Reply:
added a function to do this in pather.py and replace only the needed paths now